ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/receive/view/ReceiveView.kt

```diff
@@ -170,73 +162,139 @@ private fun ReceiveContents(
     versionInfo: VersionInfo,
     modifier: Modifier = Modifier,
 ) {
-    Column(
-        modifier =
-            Modifier
-                .fillMaxHeight()
-                .verticalScroll(rememberScrollState())
-                .then(modifier),
-        horizontalAlignment = Alignment.CenterHorizontally
-    ) {
-        if (screenBrightnessState == ScreenBrightnessState.FULL) {
-            BrightenScreen()
-            DisableScreenTimeout()
+    if (screenBrightnessState == ScreenBrightnessState.FULL) {
+        BrightenScreen()
+        DisableScreenTimeout()
+    }
+
+    val state by remember {
+        derivedStateOf {
+            listOfNotNull(
+                walletAddress.unified,
+                walletAddress.sapling.takeIf { versionInfo.isTestnet },
```

Comment:
Should we bring another UI solution instead of making the new tabs toggle three-buttons? As those surely won't fit. I propose moving the Sapling part (which is visible only in the Testnet version of the app) below the Unified one with its own title.

Reply:
Yeh I tested those - they fit because the texts wraps into 2 lines, but definitely not ideal. No figma design for this either

Reply:
But as u say, moving it to an existing tab could work